docs/user-guide/sdks-using.md

```diff
@@ -10,13 +10,41 @@ The following SDKs are available.
 
 ## Getting started
 
-To get started with the SDKs, navigate to [Zowe.org Downloads](https://www.zowe.org/download.html) and choose a programming language from the **Zowe Client SDKs** section. The package is downloaded to your computer.
+To get started, download and import the SDK packages into your project. You can also import packages from an online npm registry.
 
-<!--TODO Can we list the artifacts that the user can expect to find in the package after downloading (files, readmes, etc..)? -->
+### Install SDK from local package
 
-**Example - Node:**
+Download and install a set of offline TGZ files.
 
-Create a session object to communicate with z/OSMF. This example shows how to import the z/OS Files APIs and define your connection details in the session object:
+**Follow these steps:**
+
+1. Navigate to [Zowe.org Downloads](https://www.zowe.org/download.html) and select a programming language from the **Zowe Client SDKs** section.
+
+   The SDK is downloaded to your computer.
+
+2. Copy each TGZ file that you want to install. Paste the files into your project directory.
+3. Navigate to your project directory and issue the command `npm install <packageName>.tgz` for each package.
+
+   The packages are installed and ready to use.
+
+### Install SDK from npm registry
+
+Define packages as dependencies in your project and access them from an online npm registry.
+
+**Follow these steps:**
+
+1. Issue the following command to import a package from public npm ``.  <!--TODO unsure about this procedure-->
```

Comment:
> `<!--TODO unsure about this procedure-->`

The command to issue is `npm install <sdkPackageName>`.

Or for Python, `pip install <sdkPackageName>`.